src/Mono.Android/Android/IncludeAndroidResourcesFromAttribute.cs

```diff
@@ -3,6 +3,7 @@
 namespace Android {
 
 	[AttributeUsage (AttributeTargets.Assembly, AllowMultiple = true)]
+	[Obsolete ("This attribute is deprecated and will be removed in a future release.")]
```

Comment:
Should this include `, error: true` so that it cannot be referenced from C# code?  (I assume it can still be access via Reflection/MSBuild, but it would prevent C#-side references and more strongly signify that it's going away.)

Reply:
...and ditto all the other `[Obsolete]`s in this PR.

Reply:
Would library creations be able to ignore this error if they get it? Generally Deprecation should not result in build breakage, at least not right away.

Reply:
Doing some web searching, it appears there is not a way to get around the error at all if `IsError=True`:

https://stackoverflow.com/a/10608567/132442

They suggest using System.Reflection to workaround, which might not be possible with custom attributes. You would have to use Mono.Cecil or something.

Reply:
Discussed on Slack: we won't add `IsError=True` until we actually remove `<GetAdditional*/>`.